src/crawler/crawler/spiders/tce_downloader.py

```diff
@@ -0,0 +1,24 @@
+import rows
+from scrapy import Request, Spider
+
+
+class TCEDownloadFileSpider(Spider):
+    name = "tce-downloader"
+
+    def __init__(self, links_filename='links.csv', download_path='download/'):
+        self.links_filename = links_filename
+        self.download_path = download_path
```

Comment:
This variable is not used so can be removed (`self.crawler.settings.get('DOWNLOAD_PATH')` is used instead).